deployment/docker-compose/pretix/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:3.6
+
```

Comment:
`ENV DEBIAN_FRONTEND=noninteractive` should be added or the action starting in line 9 will silently fail. It also suppresses some errors about not specified TERM variable and interactive promts.